src/components/ListSection/ListSection.module.scss

```
@@ -0,0 +1,28 @@
+//---------------------------------------
+// Principles page
+//---------------------------------------
+.list-section {
+  --space: 24px;
+  padding-top: 2rem;
+}
+
+.list-section:before {
```

```suggestion
.list-section::before {
```

we should be able to use the CSS3 selector right?